Cargo.toml

```diff
@@ -36,19 +37,22 @@ serde = "1.0"
 serde_derive = "1.0"
 url = "2.2.2"
 reqwest = { version = "0.11", features = ["blocking", "rustls-tls", "rustls-tls-native-roots"], optional = true, default-features = false }
-picky-krb = "0.4.0"
-picky-asn1 = { version = "0.5.0", features = ["chrono_conversion"] }
-picky-asn1-der = "0.3.1"
-picky-asn1-x509 = "0.7.0"
+picky-krb = { git = "https://github.com/Devolutions/picky-rs.git", rev = "50dfc40" }
```

Comment:
I will update the `picky-*` dependencies once https://github.com/Devolutions/picky-rs/pull/186 is merged and the new versions of the crates are published

Reply:
picky crates are now published on crates.io 🙂

Reply:
@CBenoit Thanks. Now, this PR is ready to be merged 🙂